generator/index.js

```diff
@@ -46,20 +126,135 @@ module.exports.hooks = (api, options) => {
         defaultLocale: "en"
       }
     ],`)
+        }
+
+        const renderIndex3 = lines.findIndex(line => line.match(/build:/))
+        if (options.choice === 'nuxt') {
+            lines[renderIndex3] = lines[renderIndex3].replace(`build: {`, `publicRuntimeConfig: {\r
+    sections: {
+      projectId: "${options.projectId}",
```

Comment:
with cname mode this is not required anymore.. shouldn't you ask if the project is using cname or project id first?